stable/datadog/templates/container-agent.yaml

```
@@ -32,9 +32,9 @@
     {{- if .Values.datadog.nonLocalTraffic }}
     - name: DD_DOGSTATSD_NON_LOCAL_TRAFFIC
       value: {{ .Values.datadog.nonLocalTraffic | quote }}
-    - name: DD_APM_NON_LOCAL_TRAFFIC
-      value: {{ .Values.datadog.nonLocalTraffic | quote }}
     {{- end }}
+    - name: DD_APM_NON_LOCAL_TRAFFIC
```

link to another comments. I think we still need to allow people to desactivate the deployment of the trace-agent if they want.

```suggestion
    {{- if .Values.datadog.apm.enabled }}
    - name: DD_APM_NON_LOCAL_TRAFFIC
```